src/api/Folder.js

```diff
@@ -237,19 +226,17 @@ class Folder extends Item {
      * @param {Function} successCallback - Function to call with results
      * @param {Function} errorCallback - Function to call with errors
      * @param {boolean|void} [forceFetch] - Bypasses the cache
-     * @param {boolean|void} [includePreview] - Optionally include preview fields
-     * @param {boolean|void} [includePreviewSidebar] - Optionally include preview sidebar fields
+     * @param {boolean|void} [includePreviewFields] - Optionally include preview fields
+     * @param {boolean|void} [includeSidebarFields] - Optionally include preview sidebar fields
      * @return {void}
      */
-    folder(
+    getFolder(
         id: string,
         sortBy: SortBy,
         sortDirection: SortDirection,
         successCallback: Function,
         errorCallback: Function,
-        forceFetch: boolean = false,
-        includePreviewFields: boolean = false,
-        includePreviewSidebarFields: boolean = false
+        options: Object = {}
```

Comment:
Should this be typed to indicate what options are supported?

Reply:
yup yup